src/backends/torch/native/native_factory.h

```diff
@@ -44,8 +44,10 @@ namespace dd
   {
     if (tdef.find("nbeats") != std::string::npos)
       {
-        std::vector<std::string> p = template_params.get("template_params")
-                                         .get<std::vector<std::string>>();
+        std::vector<std::string> p;
```

Comment:
What happens when template params are not passed ? Shall we have a default `template_params` value here ?

Looking at the `Nbeats` constructor, the values of some parameters appear to be hardcoded into the constructor itself. 

If possible I'd put them into the class variables init directly, e.g. `int _backcast_length = 50;`.

Reply:
indeed the default template_params with this syntax is an empty vector 
will do for default values